app/dedicated-cloud-gateways/azure-peering.md

```diff
@@ -13,14 +13,109 @@ works_on:
 api_specs:
     - konnect/control-planes-config
 faqs:
-  - q: Will a new serverless gateway be provisioned in the same region as {{site.konnect_short_name}}?
-    a: Deployment on the same region is not guaranteed. 
+  - q: What is Azure VNET Peering and how does it work with {{site.konnect_short_name}}?
+    a: |
+      Azure Virtual Network Peering allows you to securely connect your Azure environment to {{site.konnect_short_name}}'s Dedicated Cloud Gateways. 
+      {{site.konnect_short_name}} hosts the Data Plane Nodes on Azure, and peering enables low-latency, secure communication between your Azure-hosted services and your {{site.konnect_short_name}} infrastructure.
```

Comment:
```suggestion
```
I think we can remove this since the intro paragraph already says it. If you remove this, make sure to get the `faqs:` too, GH won't let me suggest on that line.